pkg/manager/impl/execution_manager.go

```diff
@@ -794,6 +794,10 @@ func (m *ExecutionManager) launchExecutionAndPrepareModel(
 	if err != nil {
 		return nil, nil, err
 	}
+	rawOutputDataConfig := launchPlan.Spec.RawOutputDataConfig
```

Comment:
can you also update for single task execution too? https://github.com/flyteorg/flyteadmin/blob/master/pkg/manager/impl/execution_manager.go#L468